### PR TITLE
[Hub Generated] Review request for Microsoft.Insights to add version stable/2020-10-01

### PR DESCRIPTION
**Pull request by InbalZim**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/InbalZim/azure-rest-api-specs/dev-monitor-Microsoft.Insights-2017-04-01...master/).

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “**WaitForARMFeedback**” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/monitor/resource-manager/Microsoft.Insights/stable/2017-04-01/activityLogAlerts_API.json [hunk 1]
```diff
@@ -390,22 +396,56 @@
         "actions"
       ]
     },
+    "ActivityLogAlertCondition": {
```
Comment (MyronFanQiu) [inline specification/monitor/resource-manager/Microsoft.Insights/stable/2017-04-01/activityLogAlerts_API.json:399]: What's this used for? It seems like we only have description for this field.

Comment (MyronFanQiu) [reply specification/monitor/resource-manager/Microsoft.Insights/stable/2017-04-01/activityLogAlerts_API.json:399]: I cannot see any usage of this field too.

**Comment by MyronFanQiu** [comment]
@InbalZim Hi. Please solve CI and comments.

**Comment by MyronFanQiu** [comment]
@InbalZim Any update about this PR?

File: specification/monitor/resource-manager/Microsoft.Insights/stable/2020-10-01/activityLogAlerts_API.json [hunk 2]
```diff
@@ -0,0 +1,546 @@
...
+        "type": "oauth2",
+        "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+        "flow": "implicit",
+        "description": "Azure Active Directory OAuth2 Flow",
+        "scopes": {
+          "user_impersonation": "impersonate your user account"
+        }
+      }
+    },
+    "paths": {
```
Comment (InbalZim) [inline specification/monitor/resource-manager/Microsoft.Insights/stable/2020-10-01/activityLogAlerts_API.json:39]: @MyronFanQiu Can you please help me understand what I need to do to fix this issue?
image

image

**Comment by chiragg4u** [comment]
While adding new version the recommendation is to copy the existing version into new directory structure for first commit and then push new changes for easy review. Please create new branch with this recommendation for faster review.

**Comment by InbalZim** [comment]
> While adding new version the recommendation is to copy the existing version into new directory structure for first commit and then push new changes for easy review. Please create new branch with this recommendation for faster review.

Created a new PR as requested https://github.com/Azure/azure-rest-api-specs/pull/11579

**Pull request closed without merging**